axelrod/tests/strategies/test_calculator.py

```diff
@@ -23,28 +24,77 @@ class TestCalculator(TestPlayer):
     def test_strategy(self):
         self.first_play_test(C)
 
-        P1 = axelrod.Calculator()
-        P1.history = [C] * 20
-        P2 = axelrod.Player()
-        P2.history = [C, D] * 10
+        p1 = axelrod.Calculator()
+        p1.history = [C] * 20
+        p2 = axelrod.Player()
+        p2.history = [C, D] * 10
         # Defects on cycle detection
-        self.assertEqual(D, P1.strategy(P2))
+        self.assertEqual(D, p1.strategy(p2))
```

Comment:
on second look.  are these tests necessary?  i think i covered them in the other two test functions.  i suspect rewriting them would look exactly like the next two tests.